lib/tapioca/generator.rb

```diff
@@ -116,16 +116,26 @@ def build_todos
       say("Please review changes and commit them.", [:green, :bold])
     end
 
-    sig { params(requested_constants: T::Array[String]).void }
-    def build_dsl(requested_constants)
+    sig do
+      params(
+        requested_constants: T::Array[String],
+        should_verify: T::Boolean,
+      ).void
+    end
+    def build_dsl(requested_constants, should_verify: false)
```

Comment:
I'm thinking a bit `out loud` but happy to pair on this or talk more about it:

It feels like this method is doing too much. Like maybe it shouldn't need to know whether or not the user is verifying the files but instead maybe it could take an `outpath`, build things happily and put them wherever it's requested, and some other pieces of code that call it could be concerned with the act of verification (or otherwise), communicating with the user, etc. ya know?

Reply:
Yeah, I had those thoughts originally as well. After talking with @paracycle this was the chosen approach, and future refactors will take place to clean things up. Focus is getting the feature out in working order, than future improvements can be made to the code to tidy things up.